comments-by-guideline-and-success-criterion.md

```diff
@@ -0,0 +1,1810 @@
+
+Success Criterion 1.4.1: Use of Color (Level A)
+-----------------------------------------------
+
+From [Success Criterion 1.4.1](http://www.w3.org/TR/WCAG22/#use-of-color):
+
+Additional Guidance When Applying Success Criterion 1.4.1 to Non-Web Documents and Software:
+--------------------------------------------------------------------------------------------
+
+This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.4.1](http://www.w3.org/WAI/WCAG22/Understanding/use-of-color#intent) (also provided below).
+
+Intent from [Understanding Success Criterion 1.4.1 in Understanding WCAG 2.2](http://www.w3.org/WAI/WCAG22/Understanding/use-of-color#intent):
+
+> The intent of this Success Criterion is to ensure that all users can access information that is conveyed by color differences, that is, by the use of color where each color has a meaning assigned to it. If the information is conveyed through color differences in an image (or other non-text format), the color may not be seen by users with color deficiencies. In this case, providing the information conveyed with color through another visual means ensures users who cannot see color can still perceive the information.
+> 
+> Color is an important asset in design of Web content, enhancing its aesthetic appeal, its usability, and its accessibility. However, some users have difficulty perceiving color. People with partial sight often experience limited color vision, and many older users do not see color well. In addition, people using text-only, limited-color or monochrome displays and browsers will be unable to access information that is presented only in color.
+> 
+> Examples of information conveyed by color differences: “required fields are red", “error is shown in red", and “Mary's sales are in red, Tom's are in blue". Examples of indications of an action include: using color to indicate that a link will open in a new window or that a database entry has been updated successfully. An example of prompting a response would be: using highlighting on form fields to indicate that a required field had been left blank.
+> 
+> _Note:_ This should not in any way discourage the use of color on a page, or even color coding if it is redundant with other visual indication.
+> 
+> ###### Specific Benefits of Success Criterion 1.4.1
+> 
+> *   Users with partial sight often experience limited color vision.
+>     
```

Comment:
Bullets from the inserted Intent & Benefits seem to have an extra line between them. Will have to see if it displays that way and clean out, if necessary.